octopoes/server/server.py

```diff
@@ -1,17 +1,29 @@
 """Server that exposes API endpoints for Octopoes."""
 
 import logging
-from typing import Any, Dict
+from pathlib import Path
+from typing import Any, Dict, Optional, List
 
-import fastapi
 import uvicorn
+from fastapi import FastAPI, status
+from fastapi.responses import PlainTextResponse, HTMLResponse, JSONResponse
+from graphql import print_schema, graphql_sync
+from pydantic import BaseModel
 
 from octopoes.context.context import AppContext
 from octopoes.ingesters.ingester import Ingester
+from octopoes.models.ingester import Ingester as IngesterModel
 from octopoes.models.health import ServiceHealth
 from octopoes.version import version
 
 
+class GraphqlRequest(BaseModel):
```

Comment:
See my other point about naming and aliasing

Reply:
Fixed